nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -606,3 +601,97 @@ The module update takes care of the new config syntax and the data itself (user
 - Docker now defaults to 24, as 20.10 is stopping to receive security updates and bug fixes after [December 10, 2023](https://github.com/moby/moby/discussions/45104).
 
 - There is a new NixOS option when writing NixOS tests `testing.initrdBackdoor`, that enables `backdoor.service` in initrd. Requires `boot.initrd.systemd.enable` to be enabled. Boot will pause in stage 1 at `initrd.target`, and will listen for commands from the `Machine` python interface, just like stage 2 normally does. This enables commands to be sent to test and debug stage 1. Use `machine.switch_root()` to leave stage 1 and proceed to stage 2.
+
+## Nixpkgs library changes {#sec-release-23.11-lib}
+
+### Additions and improvements {#sec-release-23.11-lib-additions-improvements}
```

Comment:
@roberth I restructured it into just two sections instead now, one of them is here, and includes both new stuff but also improvements to stuff (intentionally not being specific what "stuff" is, so it can also include docs)